When started from shell, program doesn't terminate on Debian when closing application

The shell isn't released when OpenHantek is terminated on Debian 9.
You have to do a Ctrl-C to quit the application.

cmake fails on Debian 9:

klingler@debian:/home2/Develop/openhantek/build$ cmake ../
-- Version: openhantek2/3490680
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/Qt5/Qt5Config.cmake:26 (find_package):
  Could not find a package configuration file provided by "Qt53DCore" with
  any of the following names:

    Qt53DCoreConfig.cmake
    qt53dcore-config.cmake

  Add the installation prefix of "Qt53DCore" to CMAKE_PREFIX_PATH or set
  "Qt53DCore_DIR" to a directory containing one of the above files.  If
  "Qt53DCore" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  openhantek/CMakeLists.txt:3 (find_package)

--- a/openhantek/src/main.cpp
+++ b/openhantek/src/main.cpp
@@ -205,7 +205,10 @@ int main(int argc, char *argv[]) {
     postProcessingThread.quit();
     postProcessingThread.wait(10000);
 
-    if (context && device != nullptr) { libusb_exit(context); }
+    if (context && device != nullptr) { 
+        device.reset(); // causes libusb_close(), which must be called before libusb_exit() 
+        libusb_exit(context); 
+    }
 
     return res;
 }

